pages/configuration/server.conf.rst

```diff
@@ -305,8 +305,11 @@ Rotation
     * For a maximum of 12 hours or 5 GB whichever happens first.
     * During normal operation the journal will be smaller.
 * ``message_journal_flush_age = 1m``
+    * This setting allows specifying a time interval at which we will force an fsync of data written to the log. For example if this was set to 1000 we would fsync after 1000 ms had passed. In general we recommend you not set this and use replication for durability and allow the operating system’s background flush capabilities as it is more efficient.
```

Comment:
I would remove `In general we recommend you not set this and use replication for durability and allow the operating system’s background flush capabilities as it is more efficient.` because it is not applicable to Graylog.